components/cluster/corosync/patches/man8.patch

```diff
@@ -1077,9 +1086,58 @@ diff -Nur a/man/sam_overview.8 b/man/sam_overview.8
  
  .SH NAME
  .P
-diff -Nur a/man/votequorum.5 b/man/votequorum.5
---- a/man/votequorum.5	2017-09-26 20:23:12.074438260 +0000
-+++ b/man/votequorum.5	2017-09-26 20:25:14.945631782 +0000
+diff -Nur b/man/votequorum.5 a/man/votequorum.5
+--- b/man/votequorum.5	2019-08-19 07:35:39.871503472 +0000
++++ a/man/votequorum.5	2019-08-19 07:47:38.778202474 +0000
+@@ -61,7 +61,7 @@
+ both will disable votequorum. If both are present at the same time, 
+ the quorum.expected_votes value will override the one calculated from the nodelist.
+ .PP
+-Example (no nodelist) of an 8 node cluster (each node has 1 vote):
++Example (no nodelist) of an 1m node cluster (each node has 1 vote):
```

Comment:
Same thing here. And several other places.

Reply:
Yes, too much gsed